public/sass/_variables.dark.generated.scss

```diff
@@ -115,7 +115,7 @@ $brand-gradient-vertical: linear-gradient(#f05a28 30%, #fbca0a 99%);
 // Links
 // -------------------------
 $link-color: rgb(204, 204, 220);
-$link-color-disabled: rgba(204, 204, 220, 0.40);
+$link-color-disabled: rgba(204, 204, 220, 0.57);
```

Comment:
🤔 I think we need to add these contrast fixes on the `light` theme, I ran fast pass on light theme on org pages, and got some complains in the footer.

![fastPassOrgTeams](https://user-images.githubusercontent.com/239999/135461820-d1f1a24b-50f8-4a32-8549-4f4520e9cc0f.png)

Reply:
Ah, I use dark theme so I didn't notice that. I'll take a look, thanks!